WordPress/Classes/ViewRelated/People/PeopleViewController.swift

```diff
@@ -521,11 +521,10 @@ private extension PeopleViewController {
     }
 
     func role(person: Person) -> Role? {
-        guard let blog = blog,
-            let service = RoleService(blog: blog, context: viewContext) else {
-                return nil
+        guard let blog = blog else {
```

Comment:
I was going to suggest `guard let blog else` but I got annoyed at myself for being so pedantic about things that could be automated...

[SwiftLint supports this](https://github.com/wordpress-mobile/WordPress-iOS/issues/19549#issuecomment-1408037973), I should add the check to our https://github.com/Automattic/swiftlint-config and integrate it here...

Reply:
No worries. It's always good to have cleaner code. Updated in https://github.com/wordpress-mobile/WordPress-iOS/pull/20010/commits/441270621ae68a3ba2d00a155a9bc1082d2148b8